generator/src/main/java/org/apache/kafka/message/FieldSpec.java

```diff
@@ -89,7 +89,12 @@ public FieldSpec(@JsonProperty("name") String name,
         }
         this.fields = Collections.unmodifiableList(fields == null ?
             Collections.emptyList() : new ArrayList<>(fields));
-        this.type = FieldType.parse(Objects.requireNonNull(type));
+        this.entityType = (entityType == null) ? EntityType.UNKNOWN : entityType;
```

Comment:
Type can be either "[]int32" or "int32" if the entityType is "brokerId", how to handle this problem?

Reply:
Good question. This PR does not plan to support that cases as entity type can’t show “array” type.